effortless-chef-io/effortless-config.md

```diff
@@ -4,43 +4,51 @@ Effortless Config is the pattern for managing your Chef Infra workloads. It uses
 
 ## Patterns
 
-### Chef Repo Cookbook pattern
+### Chef Repo Cookbook Pattern
 
-This pattern is used to build policyfiles using the chef_repo pattern for organizing your cookbooks. The chef-repo pattern can be found [here](https://docs.chef.io/chef_repo/).
+This pattern builds Policyfiles using the chef-repo pattern for organizing your cookbooks. More information about the [chef-repo](https://docs.chef.io/chef_repo/) pattern can be found in our Chef Infra Documentation. View an example of this pattern [here](https://github.com/chef/effortless/tree/master/examples/effortless_config/chef_repo_pattern).
 
-> Note: You can view and example of this pattern [here](https://github.com/chef/effortless/tree/master/examples/effortless_config/chef_repo_pattern).
+1. To use this pattern, navigate to the chef-repo directory that you want to use:
 
-1. To use this pattern navigate to the chef-repo you want to use
-   ```
+   ```bash
    cd chef-repo
    ```
-1. Make a habitat directory
-   ```
+
+1. Create a `habitat` directory from the command line with:
+
+   ```bash
    mkdir habitat
    ```
+
 1. Make a plan file
-   
-   > Notes: For a cookbook targeting windows use a `plan.ps1` for Linux use a `plan.sh` if your cookbook targets both windows and linux you can have both a `plan.ps1` and a `plan.sh` in your habitat directory.
-   ```
+
+   Use a `plan.ps1` for a cookbook targeting Windows. Use a `plan.sh` for a cookbook targeting Linux. If the cookbook targets both Windows and Linux, you can have both a `plan.ps1` and a `plan.sh` in the `habitat` directory. Create a plan in Linux with the following command:
+
+   ```bash
    touch plan.sh
    ```
-1. Add some information about your cookbok to your plan
-   plan.sh
+
+1. Add information about the cookbook to the plan
+
+   Add this information to the `plan.sh` file:
+
    ```bash
-   pkg_name=<NAME FOR YOUR POLICYFILE>
-   pkg_origin=<YOUR ORIGIN>
+   pkg_name=<NAME FOR MY POLICYFILE>
+   pkg_origin=<MY ORIGIN>
    pkg_version="0.1.0"
-   pkg_maintainer="YOUR NAME AND EMAIL"
+   pkg_maintainer="MY NAME AND EMAIL"
    pkg_license=("Apache-2.0")
    pkg_scaffolding="chef/scaffolding-chef-infra"
    pkg_svc_user=("root")
-   scaffold_policy_name="<YOUR POLICYFILE NAME>"
+   scaffold_policy_name="<MY POLICYFILE NAME>"
    ```
-1. Create a policyfile directory in your chef-repo and build a policyfile
-   
-   Example of a policyfile.rb:
+
+1. Create a Policyfile directory in the chef-repo and build a Policyfile
```

Comment:
@TheLunaticScripter Is there a specific command required to build a Policyfile?

Reply:
I totally didn't realize but it looks like `chef generate policyfile`  creates a `Policyfile.rb` in the directory that the command is run in.

Reply:
Oh cool! Thanks for that! So in "building" a Policyfile, do you mean that you're adding content to the empty Policyfile?

Reply:
Yes createing the file and adding the content.

Reply:
We could reference the creation if we wanted as well:

```shell
mkdir policyfiles
chef generate policyfile policyfiles/MyPolicyfile.rb                                                                                
Recipe: code_generator::policyfile
  * template[/workspace/my_repo/policyfiles/MyPolicyfile.rb] action create
    - create new file /workspace/my_repo/policyfiles/MyPolicyfile.rb
    - update content in file /workspace/my_repo/policyfiles/MyPolicyfile.rb from none to fc117f
    (diff output suppressed by config)
```

Reply:
```suggestion
1. If you do not have a policyfiles directory in your chef-repo, create one:
 
   ```
   mkdir policyfiles
   ```

1. Generate a Policyfile

  ```
  chef generate policyfile policyfiles/my_policyfile
  ```
```

Reply:
Does chef generate `my_policyfile` require the `.rb` extension?

Reply:
it does not require the `.rb` extension